frontend/src/components/ConfirmDelete.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Modal, Button} from 'react-bootstrap';
+import { fb } from '../App';
+
+export default class ConfirmDelete extends Component {
+  handleDelete() {
+    // delete event from entire database
+    fb.eventsRef.child("events").child(this.props.reference).remove();
+
+    //delete event from user's table
+    fb.userRef.child("events").child(this.props.uid).child(this.props.reference).remove();
```

Comment:
I don't know if you can delete events from anywhere else, but you may want to put this in some sort of helper function so that you can easily just call `deleteEvent(this.props.reference)` and not worry about keeping events and users in sync

Reply:
that was for testing, john has the code in the firebase.js. we just make a request from there